src/main/java/frc/robot/subsystems/led/Led2023.java

```diff
@@ -122,28 +147,65 @@ public Led2023(Arm arm, IntakeRelease intakerelease) {
   public void resetTimers() {
     rainbowTimer.reset();
     purpleTimer.reset();
```

Comment:
rename this to victoryTimer.

Reply:
I meant to create a new timer, will need to change this because purple timer is being used for something else.